tests/api/test_picking.py

```diff
@@ -292,6 +292,7 @@ def test_build_primers_amplicon_size_filtering(
         assert pair.amplicon.end == pair.right_primer.span.end
         assert pair.bases == REF_BASES[pair.amplicon.start - 1 : pair.amplicon.end]
         assert pair.span.length <= 200
+        assert pair.span.length >= 100
```

Comment:
**suggestion** Add an explicit test on the cardinality of `pairs`, rather than requiring it be non-empty?

I think we can calculate the number of pairs we expect to receive from `build_primer_pairs()`.

Reply:
I'd prefer not to do this ... mostly because it would make the test more brittle, and any change to the test would require either re-calculating by hand (or worse, running the test and then just copying the number over as expected).